.github/workflows/windows-e2e.yaml

```diff
@@ -0,0 +1,51 @@
+name: e2e tests on Windows
+
+on:
+  schedule:
+    - cron: '13 5 * * 1-5'
+  workflow_dispatch: {}
+defaults:
+  run:
+    shell: powershell
+jobs:
+
+  e2e-tests:
+    timeout-minutes: 90
+    runs-on: [self-hosted, Windows, X64]
```

Comment:
please, add label "win11"

Reply:
I can do that, sure, but note that nothing in this workflow actually requires Windows 11 (as opposed to Windows 10). So that give me pause, as that means we're using the "win11" label to (very opaquely) filter on something _else_.